superset-frontend/src/dashboard/components/SliceAdder.jsx

```diff
@@ -240,6 +271,18 @@ class SliceAdder extends React.Component {
       MARGIN_BOTTOM;
     return (
       <div className="slice-adder-container">
+        <NewChartButtonContainer>
+          <NewChartButton
+            buttonStyle="link"
+            buttonSize="xsmall"
+            onClick={() =>
+              window.open('/chart/add', '_blank', 'noopener noreferrer')
```

Comment:
It could be interesting to implement broadcasting so that the list is updated when a chart is created the chart list is refreshed when the new chart is saved: https://developer.mozilla.org/en-US/docs/Web/API/Broadcast_Channel_API

Reply:
Good idea! But wouldn't we actually need to use websocket instead to update the list when chart is saved on the backend? I just wonder if such frontend communication would be reliable enough